wordSearch.c

```diff
@@ -73,29 +88,36 @@ int main(void)
 
     fprintf(fp, "</p>");
     fprintf(fp, "</table></body></html>");
+
     fclose(fp);
+    fclose(wordFile);
 
     return 0;
 }
 
-void wordPick(FILE *fp, char board[DIM_X][DIM_Y])
+void loadWords(FILE *wordFile, char words[FILE_WORD_COUNT][MAX_WORD_LENGTH])
+{
+    int wordsRead = 0;
+    for (int i = 0; fgets(words[i], BUFSIZ, wordFile); i++) {
+        wordsRead++;
+    }
+    if (wordsRead != FILE_WORD_COUNT) {
+        fprintf(stderr, "Reading error. Expected %d words but got %d\n",
+            FILE_WORD_COUNT, wordsRead);
+        exit(1);
+    }
+}
```

Comment:
this is actually where i realized the file only had 2999 words. cause i added an integrity check that made sure exactly `FILE_WORD_COUNT` words were read in